### DISCUSSION TITLE
Link to O2M & M2O fields when they are disabled for editing

### DISCUSSION THREAD

**Discussion by goggi** [directus/directus#3033] [Feature Requests]
It would be great if we could have a link to O2M&M2O fields items even if the fields are disabled.

![image](https://user-images.githubusercontent.com/1946339/96710945-118a4680-139d-11eb-99ab-08e74be65df7.png)

So the link could be at the yellow part, and that would take you directly to the item or maybe open the modal with the item?

**Comment by rijkvanzanten** [comment]
This now exists!
